client/src/main/java/com/battleships/LogMessages.java

```diff
@@ -0,0 +1,13 @@
+package com.battleships;
+
+public class LogMessages {
+    public static final String DISCONNECTED_AFTER_PLAYER_REQ = "Rozłączony z serwerem na żądanie gracza";
```

Comment:
What about enum with values? https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html (bottom of the page).

Reply:
We prefer static Strings instead of making trains like LogMessages.DISCONNECTED_AFTER_PLAYER_REQ.toString()

If you have better idea please let us know